modules/capability/genesis_test.go

```diff
@@ -1,21 +1,18 @@
 package capability_test
 
 import (
-	dbm "github.com/cometbft/cometbft-db"
-	"github.com/cometbft/cometbft/libs/log"
-	tmproto "github.com/cometbft/cometbft/proto/tendermint/types"
-	simtestutil "github.com/cosmos/cosmos-sdk/testutil/sims"
-	sdk "github.com/cosmos/cosmos-sdk/types"
 	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	stakingtypes "github.com/cosmos/cosmos-sdk/x/staking/types"
-	"github.com/cosmos/ibc-go/v7/testing/simapp"
 
 	"github.com/cosmos/ibc-go/modules/capability"
 	"github.com/cosmos/ibc-go/modules/capability/keeper"
 	"github.com/cosmos/ibc-go/modules/capability/types"
 )
 
 func (suite *CapabilityTestSuite) TestGenesis() {
+	// InitGenesis must be called in order to set the intial index to 1.
+	capability.InitGenesis(suite.ctx, *suite.keeper, *types.DefaultGenesis())
```

Comment:
`InitGenesis` has to be called directly here in order to set the initial capability index to 1. 

This used to be happening handled implicitly when testing with a simapp. This caught me off guard for a while as after removing simapp this test would fail below when trying to lookup the first capability after genesis is reinitialised. 

The reasoning for this is that without setting the initial index to 1 - `cap1` created on line 19 would be at index 0, and `ExportGenesis` starts the loop index [here](https://github.com/cosmos/ibc-go/blob/main/modules/capability/genesis.go#L31) at 1. So after exporting and then reinitialising the gen state we'd lose first capabiliity.